at-luantran/Week5/migrate/20180717095550_change_gender_to_be_int_in_users.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeGenderToBeIntInUsers < ActiveRecord::Migration[5.2]
+  def up
+    User.where(gender: 'nam').update_all(gender: '1')
+    User.where(gender: 'nu').update_all(gender: '2')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/b6e75bf50d420fce7e508480ec5e6e3bea68c53e/at-luantran/Week5/migrate/20180717095550_change_gender_to_be_int_in_users.rb#L4"></span>Rails/SkipsModelValidations: Avoid using <code>update_all</code> because it skips validations. (http://guides.rubyonrails.org/active_record_validations.html#skipping-validations)</td>
    </tr>
  </tbody>
</table>